setup.py

```diff
@@ -24,6 +24,7 @@
 install_requires = [
     "requests>=2.7,<3.0",
     "six>=1.5",
+    "mock>=2.0.0",
```

Comment:
Can we make this a test only dependency? We don't want our users to have to pull in `mock` to the use the analytics-python package.